controllers/interest.js

```diff
@@ -0,0 +1,52 @@
+const Interest = require('../models/interest')
+
+function index(req, res, next) {
+  Interest.find({})
+    .exec()
+    .then((interest) => res.json(interest))
+    .catch((err) => next(err))
+}
+
+function create(req, res, next) {
+  const { name } = req.body
+
+  Interest.create({ name })
+    .then((interest) => res.json(interest))
+    .catch((err) => next(err))
+}
+
+function read(req, res, next) {
+  const { id } = req.params
+
+  Interest.findById(id)
+    .exec()
+    .then((interest) => res.json(interest))
+    .catch((err) => res.sendStatus(400))
```

Comment:
Wasn't there supposed to be a global error handler (in routes/api/index.js)? We could instead do
```js
(err) => {
  err.status = 400 // perhaps a 404
  next(err)
}
```